### PR TITLE
Ntl

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
I've cancelled appveyor because windows is skipped. Otherwise, tests pass.
Ready for review

**Comment by isuruf** [comment]
@ocefpaf, ping

File: recipes/gf2x/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://gforge.inria.fr/frs/download.php/file/30873/{{ name }}-{{ version }}.tar.gz
+  sha256: 0d3f01604680102a00ca34e079903cc4d5a3208afda223748979b724d358849f
+  patches:
+    # patches from https://github.com/sagemath/sage
```
Comment (ocefpaf) [inline recipes/gf2x/meta.yaml]: :+1:

File: recipes/ntl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 0
+  skip: true        # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libtool
+    - automake
+    - gf2x
```
Comment (ocefpaf) [inline recipes/ntl/meta.yaml]: Some story about the race condition. Remember to restart this one when `gf2x` is available.

**Comment by isuruf** [comment]
Aargh, it's been 3 days and the feedstock is still not complete.

**Pull request merged**